tests/kuttl/tests/operator-upgrade/00-assert.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-local-storage-operator
+  namespace: spoke1
+spec:
+  scope:
+    name: local-storage-operator
+    namespace: openshift-local-storage
+    resource: subscriptions.operators.coreos.com
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-local-storage-operator
+  namespace: spoke2
+spec:
+  scope:
+    name: local-storage-operator
+    namespace: openshift-local-storage
+    resource: subscriptions.operators.coreos.com
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-local-storage-operator
+  namespace: spoke5
+spec:
+  scope:
+    name: local-storage-operator
+    namespace: openshift-local-storage
+    resource: subscriptions.operators.coreos.com
+---
+# Check for the PAO operator subscription.
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-performance-addon-operator
+  namespace: spoke1
+spec:
+  scope:
+    name: performance-addon-operator
+    namespace: openshift-performance-addon-operator
+    resource: subscriptions.operators.coreos.com
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-performance-addon-operator
+  namespace: spoke2
+spec:
+  scope:
+    name: performance-addon-operator
+    namespace: openshift-performance-addon-operator
+    resource: subscriptions.operators.coreos.com
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-performance-addon-operator
+  namespace: spoke5
+spec:
+  scope:
+    name: performance-addon-operator
+    namespace: openshift-performance-addon-operator
+    resource: subscriptions.operators.coreos.com
+---
+# Check for the PTP operator subscription.
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-ptp-operator-subscription
+  namespace: spoke1
+spec:
+  scope:
+    name: ptp-operator-subscription
+    namespace: openshift-ptp
+    resource: subscriptions.operators.coreos.com
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-ptp-operator-subscription
+  namespace: spoke2
+spec:
+  scope:
+    name: ptp-operator-subscription
+    namespace: openshift-ptp
+    resource: subscriptions.operators.coreos.com
+---
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: cgu-default-subscription-ptp-operator-subscription
+  namespace: spoke5
+spec:
+  scope:
+    name: ptp-operator-subscription
+    namespace: openshift-ptp
+    resource: subscriptions.operators.coreos.com
+---
```

Comment:
Looks like sriov-network-operator-subscription MCVs for spoke2 and spoke5 are missing